Shoko.Server/API/v3/Models/Shoko/Episode.cs

```diff
@@ -119,7 +119,7 @@ public class Episode : BaseModel
     [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
     public IEnumerable<FileCrossReference.EpisodeCrossReferenceIDs>? CrossReferences { get; set; }
 
-    public Episode(HttpContext context, SVR_AnimeEpisode episode, HashSet<DataSource>? includeDataFrom = null, bool includeFiles = false, bool includeMediaInfo = false, bool includeAbsolutePaths = false, bool withXRefs = false)
+    public Episode(HttpContext context, SVR_AnimeEpisode episode, HashSet<DataSource>? includeDataFrom = null, bool includeFiles = false, bool includeMediaInfo = false, bool includeAbsolutePaths = false, bool withXRefs = false, bool includeReleaseInfo = false)

```

Comment:
I think we need to rethink this API model a bit better.
Something like

```cs
new Episode(_httpContext, _episode)
  .IncludeDataFrom(...)
  .WithFiles()
  .WithMediaInfo()
  .AbsolutePaths()
  .XRefs()
  .WithReleaseInfo()
```
might reduce some of the constructor bloat.

Reply:
It's not a bad idea to clean up the constructor bloat in the API layer, but can we work on this after this PR? I can look into it unless you or somebody else wants to after the PR, but I don't want to deal with converting it _just now_.